urllib3/connection.py

```diff
@@ -195,10 +195,17 @@ class VerifiedHTTPSConnection(HTTPSConnection):
     Based on httplib.HTTPSConnection but wraps the socket with
     SSL certification.
     """
-    cert_reqs = None
     ca_certs = None
     assert_fingerprint = None
 
+    @property
```

Comment:
Fixed in eee6c87, [Travis build](https://travis-ci.org/divergentdave/urllib3/builds/64232898)